alias

```diff
@@ -134,3 +134,19 @@ revoke-my-ip-all =
   !f() {
     aws revoke-my-ip ${1} all all
   }; f
+
+s3cat =
+  !f() {
+    tmpfile=$(mktemp)
+    aws s3 cp $1 $tmpfile
```

Comment:
Pro-tip: You can copy a file to stdout using `aws s3 cp $1 -` instead of having to use temp files